aries-site/src/examples/components/pagination/PaginationServerTableExample.js

```diff
@@ -104,35 +116,50 @@ export const PaginationServerTableExample = () => {
       </Heading>
 
       <Box>
-        <DataTable
-          aria-describedby="server-side-pagination-heading"
-          columns={columns}
-          data={data}
-          sort={{ ...sort, external: true }}
-          step={limit}
-          onSort={opts => setSort(opts)}
-          fill
-          gap="small"
-        />
+        <Box overflow="auto">
+          <DataTable
+            aria-describedby="server-side-pagination-heading"
+            columns={columns}
+            data={data}
+            sort={{ ...sort, external: true }}
+            onSort={opts => setSort(opts)}
+            fill
+            gap="small"
+          />
+        </Box>
 
         {numberItems > limit && (
           <Box
             direction="row-responsive"
-            fill="horizontal"
             border="top"
-            justify="end"
-            pad={{ vertical: 'xsmall' }}
+            gap="medium"
+            pad={{ vertical: 'small' }}
+            align="center"
+            wrap
```

Comment:
```suggestion
            justify="between"
            wrap
```
This will make sure any extra space is put between the "Show X of Y" and the group containing the select + pagination.